Dockerfile

```diff
@@ -44,6 +44,9 @@ RUN curl -fsSL $pycharm_source -o /opt/pycharm/installer.tgz \
   && tar --strip-components=1 -xzf installer.tgz \
   && rm installer.tgz
 
+RUN echo "Fix for some type annotations, for example in the redis package" \
```

Comment:
Thanks for the PR! How about placing instructions and advices that are not part of the build process in the [README.md](https://github.com/SUNET/pycharm-docker/blob/main/README.md) instead?